ddio/sdlcontroller.cpp

```diff
@@ -1401,7 +1401,7 @@ int CTLLex(const char *command) {
 void sdlgameController::parse_ctl_file(int devnum, const char *ctlname) {
   // parse each file until we find a name match, no name match, just return
   ddio_DoForeachFile(
-      Base_directory, std::regex(".*\\.ctl"), [this, &devnum, &ctlname](const std::filesystem::path &path) {
+      cf_GetWritableBaseDirectory(), std::regex(".*\\.ctl"), [this, &devnum, &ctlname](const std::filesystem::path &path) {
```

Comment:
Files can be not only in write-path.

Reply:
OK. I changed that code so that it looks for `.ctl` files in each of the base directories. In order to do so, I added code to `ddio/sdlcontroller.cpp` that accesses the `Base_directories` variable directly. Is that OK?

Reply:
Yes, that OK.